src/VisualStudio/Core/Impl/SolutionExplorer/DiagnosticItem/SourceGeneratorItem.cs

```diff
@@ -13,18 +13,18 @@ internal sealed partial class SourceGeneratorItem : BaseItem
     {
         public ProjectId ProjectId { get; }
 
-        public string GeneratorAssemblyName { get; }
+        public SourceGeneratorIdentity Identity { get; }
 
         // Since the type name is also used for the display text, we can just reuse that. We'll still have an explicit
         // property so the assembly name/type name pair that is used in other places is also used here.
         public string GeneratorTypeName => base.Text;
```

Comment:
Is this still used?

Reply:
Oh probably not, great catch.

Reply:
Fixed.

Reply:
Turns out it _was_ used, didn't realize there was a partial there. Fixed that too.